setup.py

```diff
@@ -8,4 +8,5 @@
                         'pyyaml',
                         'pTable',
                         'matplotlib',
-                        'mpi4py'])
+                        'mpi4py',
+                        'ray[rllib]'])
```

Comment:
Is it a good idea to make MADRaS dependent on RLLib? I suggest we remove this dependency.

Reply:
I too think that dependence on RLLib will cause troubles. I'll try and find a workaround.